notebooks/demo_update_metadata.ipynb

```diff
@@ -41,12 +41,12 @@
     "print(os.getcwd())\n",
     "updates = \"file:///{}\".format(os.path.join(os.getcwd(), \"metadata-conversion/simple-conversion.yaml\"))\n",
```

Comment:
This is my fault to include "file:///" in front of the file path. The correct file path assignment would be:
```
updates = os.path.join(os.getcwd(), "metadata-conversion/simple-conversion.yaml")
```
My bad.